frontend/src/logic/schedule-parser/schedule.parser.ts

```diff
@@ -1,29 +1,32 @@
 import { StringHelper } from "../../helpers/string.helper";
 import { WorkerType } from "../../state/models/schedule-data/employee-info.model";
-import { ScheduleDataModel } from "../../state/models/schedule-data/schedule-data.model";
 import { ChildrenInfoParser } from "./children-info.parser";
 import { DataRowParser } from "./data-row.parser";
 import { MetaDataParser } from "./metadata.parser";
 import { ShiftsInfoParser } from "./shifts-info.parser";
+import { ScheduleProvider, Schedule } from "../schedule-provider";
 
-export class ScheduleParser {
+export class ScheduleParser implements ScheduleProvider {
   //#region members
-  private nurseShiftsParser: ShiftsInfoParser;
```

Comment:
Ze względu na użycie interfejsu musiałem zmienić poziom dostępu. Z tego co znalazłem to najbezpieczniejsze jest readonly. Chyba, że znacie jakiś lepszy trick? @Qwebeck @Sig00rd